### PR TITLE
Cmr 4001

### PR DESCRIPTION
**Pull request by mschmele**
These are some changes based on further testing both locally and in WL that address a handful of small issues that were found, as well as updating tests to cover those cases

### CODE REVIEW

File: system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj [hunk 1]
```diff
@@ -88,17 +90,26 @@
     (testing "Old and deleted collections should not be found."
       (let [range-references (search/find-concepts-with-param-string
                                "collection"
-                               "has_granules_created_at=2014-01-01T10:00:00Z,2016-02-01T10:00:00Z")
+                               "has_granules_created_at=[]=2014-01-01T10:00:00Z,2016-02-01T10:00:00Z")
```
Comment (daniel-zamora) [inline system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj]: is the param string suppose to be word=[]=value?

File: search-app/src/cmr/search/services/query_service.clj [hunk 2]
```diff
@@ -271,23 +273,56 @@
...
+        results (qe/execute-query context query)
+        collection-ids (get-bucket-values-from-aggregation results :collections)
+        collection-search-params (-> params
+                                     (assoc :echo-collection-id collection-ids)
+                                     (dissoc :has_granules_created_at)
+                                     (dissoc :has-granules-created-at)
+                                     lp/process-legacy-psa)]
+
+    (if (empty? collection-ids)
+      {:results (common-search/search-results->response context query results)
```
Comment (chris-durbin) [inline search-app/src/cmr/search/services/query_service.clj]: This isn't returning the time taken apparently.

File: system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj (hunk 1 above)
Comment (mschmele) [inline system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj]: For temporal ranges, I think so yes

File: system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj [hunk 3]
```diff
@@ -19,6 +19,8 @@
                       [(ingest/reset-fixture {"provguid1" "PROV1"})
                        (dev-system-util/freeze-resume-time-fixture)]))
 
+(def empty-response-body "<?xml version=\"1.0\" encoding=\"UTF-8\"?><results><hits>0</hits><took></took><references></references></results>")
```
Comment (chris-durbin) [inline system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj]: Took should have how long the request took.

File: system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj (hunk 1 above)
Comment (chris-durbin) [inline system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj]: No...
it should be has_granules_created_at[]=2014...

**Comment by mschmele** [comment]
Chris is going to incorporate some of this into his upcoming pull request

**Pull request closed without merging**